src/Orchard.Web/Modules/Orchard.ImportExport/Services/SetupService.cs

```diff
@@ -101,6 +102,10 @@ public class SetupService : Component, ISetupService {
                     schemaBuilder.AlterTable("Orchard_Framework_DataMigrationRecord",
                         table => table.AddUniqueConstraint("UC_DMR_DataMigrationClass_Version", "DataMigrationClass", "Version"));
```

Comment:
Yes, that's what I mean. A single class whose responsibility is to create all the schema stuff that for one reason or another can't be created through migrations.

And btw, you are not @paynecrl97... :)